Emerald.App/Emerald.App/Emerald.App.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net6.0-windows10.0.22000.0</TargetFramework>
+	  <TargetFramework>net7.0-windows10.0.22000.0</TargetFramework>
```

Comment:
@NoobNotFound this may have caused a problem..
I managed to fix AZP, but
![image](https://user-images.githubusercontent.com/71598437/216758027-3e20e681-20ab-4c87-b77c-665ba1be0f53.png)
[`Riverside.Depth.Emerald(_build, runs/11083280067)`](https://github.com/RiversideValley/Emerald/runs/11083280067)

Reply:
let me see

Reply:
You can build locally?

Reply:
Um I removed net472 from frameworks and yes I can build locally